api/types/types.go

```diff
@@ -0,0 +1,11 @@
+package types
+
+// ContainerCreateResponse contains the information returned to a client on the
+// creation of a new container.
+type ContainerCreateResponse struct {
+	// ID is the id of the created container.
+	ID string `json:"Id"`
```

Comment:
Explicit is better than implicit especially when we want to enforce API consistency and make sure that the tags don't change the responses.